src/kirby/components/icon/icon.component.tns.global.scss

```diff
@@ -1,9 +1,19 @@
 @import '../../scss/native-imports';
- 
+
 .kirby-icon {
+    display: none;
     Label {
         font-family: "Ionicons", "ionicons";
         text-align: center;
+
+        &.outline {
+            // Inherit or currentColor is not supported by native script :(
+            // TODO: need to figure out a better solution for setting border-color
+            border-color: get-color('dark');
+            border-width: 3;
+            padding: size('m');
+            border-radius: 50%;
```

Comment:
Consider using a large DIP value instead - percentage won't give you equally rounded corners unless the border-box is squared (same width/height)